tests/baselines/reference/docker/vue-next.log

```diff
@@ -1,227 +1,141 @@
-Warning: dist/packages/shared/src/toDisplayString.d.ts:2:37 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
-Warning: dist/packages/shared/src/toDisplayString.d.ts:2:19 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-Warning: dist/packages/shared/src/toDisplayString.d.ts:2:20 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-/vue-next/packages/size-check/src/index.ts → packages/size-check/dist/size-check.global.js...
-created packages/size-check/dist/size-check.global.js in ?s
-/vue-next/packages/size-check/src/index.ts → packages/size-check/dist/size-check.global.prod.js...
-created packages/size-check/dist/size-check.global.prod.js in ?s
-/vue-next/packages/template-explorer/src/index.ts → packages/template-explorer/dist/template-explorer.global.js...
-created packages/template-explorer/dist/template-explorer.global.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.esm-bundler.js...
-created packages/vue/dist/vue.esm-bundler.js in ?s
-/vue-next/packages/vue/src/runtime.ts → packages/vue/dist/vue.runtime.esm-bundler.js...
-created packages/vue/dist/vue.runtime.esm-bundler.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.cjs.js...
-created packages/vue/dist/vue.cjs.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.global.js...
-created packages/vue/dist/vue.global.js in ?s
-/vue-next/packages/vue/src/runtime.ts → packages/vue/dist/vue.runtime.global.js...
-created packages/vue/dist/vue.runtime.global.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.esm-browser.js...
-created packages/vue/dist/vue.esm-browser.js in ?s
-/vue-next/packages/vue/src/runtime.ts → packages/vue/dist/vue.runtime.esm-browser.js...
-created packages/vue/dist/vue.runtime.esm-browser.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.cjs.prod.js...
-created packages/vue/dist/vue.cjs.prod.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.global.prod.js...
-created packages/vue/dist/vue.global.prod.js in ?s
-/vue-next/packages/vue/src/runtime.ts → packages/vue/dist/vue.runtime.global.prod.js...
-created packages/vue/dist/vue.runtime.global.prod.js in ?s
-/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.esm-browser.prod.js...
-created packages/vue/dist/vue.esm-browser.prod.js in ?s
-/vue-next/packages/vue/src/runtime.ts → packages/vue/dist/vue.runtime.esm-browser.prod.js...
-created packages/vue/dist/vue.runtime.esm-browser.prod.js in ?s
+[!] (plugin rpt2) Error: /vue-next/packages/reactivity/src/baseHandlers.ts(192,3): semantic error TS2322: Type '(target: object) => (string | number | symbol)[]' is not assignable to type '(target: object) => ArrayLike<string | symbol>'.
```

Comment:
error is correct; vue's `function ownKeys` should return `(string | symbol)[]` not `(string | number | symbol)[]`. This is a breaking change for 4.2 from #41987.

@DanielRosenwasser do we normally document lib or DOM breaks in the release notes? I don't think we do.